### ISSUE TITLE
can't install version 3.2.x with python2

### ISSUE THREAD

**Issue by leifliddy** [Python-Markdown/markdown#929]
I'm not able to install ```version 4.2.1```, due to ```importlib.util``` not being found.
I'm not sure exactly which module I need to install to meet this dependency. 
I didn't experience this issue with ```version 3.1.x```

```
[root@black markdown-3.2.1]# python2.7 setup.py 
Traceback (most recent call last):
  File "setup.py", line 39, in <module>
    __version__, __version_info__ = get_version()
  File "setup.py", line 32, in get_version
    import importlib.util
ImportError: No module named util
```

**Comment by waylan** [comment]
As noted in the [release notes][1], we dropped support for Python 2.7 in version 3.2. You have two options:

1. Update to Python 3.5 or later.
2. Continue to use Python-Markdown 3.1 with Python 2.7.

I suppose we could test and add an error message to the `setup.py` script. We did include an error message for those who have a `develop` install of the package. And pip will refuse to install 3.2 on Python 2.7 due to the meta-data mismatch.

[1]: https://python-markdown.github.io/change_log/release-3.2/#drop-support-for-python-27
